index.html

```diff
@@ -1,11 +1,582 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Page under construction</title>
```

Comment:
keep eeverything in home for right now. we need to wait for dr. raskar's review before moving to index.htlm